sdk/tables/Azure.Data.Tables/tests/TableEntityTests.cs

```diff
@@ -59,14 +59,27 @@ public void DynamicEntityGetWrongTypesThrows()
             Assert.That(() => entityWithAllTypes.GetInt64("binary"), Throws.InstanceOf<InvalidOperationException>(), "GetInt64 should validate that the value for the inputted key is a Int64.");
         }
 
+        /// <summary>
+        /// Validates getters for non-nullable types involving null. (ex. DateTime, int)
+        /// </summary>
+        [Test]
+        public void ValidateDictionaryTableEntityGetNonNullType()
+        {
+            var dummyKey = "I'm a little dumb";
+            Assert.That(entityWithAllTypes.GetBoolean(dummyKey), Is.Null);
+            Assert.That(entityWithAllTypes.GetDateTime(dummyKey), Is.Null);
+            Assert.That(entityWithAllTypes.GetDouble(dummyKey), Is.Null);
+            Assert.That(entityWithAllTypes.GetInt32(dummyKey), Is.Null);
+            Assert.That(entityWithAllTypes.GetInt64(dummyKey), Is.Null);
```

Comment:
Wouldn't these all throw `InvalidOperationException`?

Reply:
The `dummyKey` was supposed to be the key for a property that doesn't exist! And for now, I've made `TableEntity` return null for either null or nonexistent properties.